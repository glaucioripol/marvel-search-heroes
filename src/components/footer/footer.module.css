.footer {
  background-color: var(--red-5);
  position: relative;
  height: 80px;
  margin-block-start: 10rem;
  width: 100%;
}

.footer p {
  color: #fff;
  font-size: var(--font-x-sm);

  position: absolute;
  bottom: 16px;
  right: 16px;
}
