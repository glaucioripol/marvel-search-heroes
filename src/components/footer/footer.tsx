import styles from "./footer.module.css";

export function Footer() {
  return (
    <footer className={styles.footer}>
      <p>Feito com ❤️ por Gláucio</p>
    </footer>
  );
}
