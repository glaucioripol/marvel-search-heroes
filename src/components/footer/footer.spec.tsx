import {
  defaultRender,
  screen,
  beforeAll,
  describe,
  expect,
  it,
} from "@/__tests__";
import { Footer } from "./footer";

describe("Footer component", () => {
  beforeAll(() => {
    defaultRender(<Footer />);
  });

  it("should render without crashing", () => {
    expect(screen.getByText(/Feito com ❤️ por Gláucio/i)).toBeInTheDocument();
  });

  it("should match snapshot", () => {
    expect(screen).toMatchSnapshot();
  });
});
