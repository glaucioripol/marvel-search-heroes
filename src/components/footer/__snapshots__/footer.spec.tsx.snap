// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > should match snapshot 1`] = `<DocumentFragment />`;
